app/controllers/proposals_controller.rb

```diff
@@ -14,10 +14,9 @@ def new
 
   def create
     if user_signed_in?
-      @candidate = Candidate.where(user_id: current_user.id).first
-      @candidature = Candidature.where(candidate_id: @candidate.id).first
+      @representative = Representative.where(user_id: current_user.id).first
```

Comment:
Hum, entendi. Não lembro bem qual era a minha duvida à época em que escrevi o comentário. Acho que devia ser algo relacionado à utilização dessa variável na view. Ela é usada na view? Se não for, pode ser só `representative` em vez de `@representative`, né?